core/src/main/java/com/exadel/kaliada/core/servlets/LikeComponentServlet.java

```diff
@@ -0,0 +1,84 @@
+/*
+ *  Copyright 2015 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.exadel.kaliada.core.servlets;
+
+import org.apache.jackrabbit.commons.JcrUtils;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.servlets.SlingAllMethodsServlet;
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet;
+import org.apache.sling.servlets.annotations.SlingServletPaths;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.propertytypes.ServiceDescription;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.jcr.*;
+import javax.servlet.Servlet;
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.lang.module.ResolutionException;
+
+/**
+ * Servlet that writes some sample content into the response. It is mounted for
+ * all resources of a specific Sling resource type. The
+ * {@link SlingSafeMethodsServlet} shall be used for HTTP methods that are
+ * idempotent. For write operations use the {@link SlingAllMethodsServlet}.
+ */
+@Component(service = { Servlet.class })
+@SlingServletPaths(value = "/bin/like")
```

Comment:
These two Apache annotations are working but somewhat obsolete.
We usually go with the only annotation:
```
@Component(
    service = Servlet.class,
    property = {
        "sling.servlet.paths=/apps/my-project/something",
        "sling.servlet.methods=" + HttpConstants.METHOD_GET // optional unless the servlet is POST-serving
    })
```
etc.
Things like the label and descriptions are needed only if a service/servlet has manually configurable properties. Then, they are seen in config page. Otherwise, they are nowhere to be seen. Therefore, they are only specified if a config interface is specified (see: OCD config)